composer.json

```diff
@@ -143,7 +144,10 @@
 		],
 		"ci:php:cs-fixer": "php-cs-fixer fix --config .php-cs-fixer.php -v --dry-run --diff",
 		"ci:php:lint": "find .*.php *.php Classes Configuration Tests -name '*.php' -print0 | xargs -r -0 -n 1 -P 4 php -l",
-		"ci:php:sniff": "phpcs Classes Configuration Tests",
+		"ci:php:sniff": [
+			"phpcs --config-set installed_paths vendor/phpcompatibility/php-compatibility",
```

Comment:
This needs to be `.Build/vendor/…`.

Reply:
According to the docs, it's recommended to use a Composer plugin to register the additional sniffs. https://github.com/PHPCompatibility/PHPCompatibility?tab=readme-ov-file#installation-in-a-composer-project-method-1